core/src/components/refresher/refresher.tsx

```diff
@@ -396,6 +396,8 @@ export class Refresher implements ComponentInterface {
       return;
     }
 
+    await contentEl.componentOnReady();
```

Comment:
If we add this here, then we should remove the `componentOnReady` call in https://github.com/ionic-team/ionic-framework/blob/master/core/src/components/refresher/refresher.tsx#L374 as it is redundant.